Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.11
+FROM python:3.19
```

Comment:
Can you confirm this is right? I thought [3.12.4](https://docs.python.org/release/3.12.4/) was the latest Python version

Reply:
Ooops, not sure how I ended up with that image tag. I've fixed it in [ef6560b](https://github.com/advanced-security/ghas-reviewer-app/pull/70/commits/ef6560ba5c769406c92982b0833bb062628f3bf3) by using `python:alpine3.19`.

I saw the issue #1, so I thought this could be one way to address it. Using this image brings down the total image size to ~130MB, from the 1.1GB when using `python:3` tag. The image will run python version 3.12.5.